ts/Dashboards/Plugins/HighchartsComponent.ts

```diff
@@ -993,8 +994,8 @@ namespace HighchartsComponent {
          */
         chartID?: string;
         /**
-         * Names / aliases that should be mapped to xAxis values. You can use
-         * null to keep columns selectively out of the chart.
+         * Names / aliases that should be mapped to xAxis values.
```

Comment:
Sorry for being picky but I would describe how it works. What is going to happen if you don't declare them etc.